feature/apartment/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="open">Open</string>
```

Comment:
please be more descriptive in all the strings name like: 
`text_status_open`
`text_title_information`
`text_button_history`
`text_button_plus_new`

Reply:
Done